scripts/gen_oscal.py

```diff
@@ -91,10 +124,9 @@ def generate_models():
     out_dir.mkdir(exist_ok=True, parents=True)
     tmp_dir = out_dir / 'tmp'
     tmp_dir.mkdir(exist_ok=True, parents=True)
-    out_init = out_dir / '__init__.py'
-    out_init.touch(exist_ok=True)
-
-    in_dir = Path('nist-source/json/schema')
+    src_dir = 'release-1.1.2-schemas'
```

Comment:
I think this shoudl be reverted to be dynamic unless a clear reason exists to commit this into the project.

Reply:
Fixed. 

- Use `release-schemas` folder for raw schemas used (e.g., remove version 1.1.2)
-  Use tmp folder for fixup schemas (e.g., remove from trestle repo)